internal/handlers/handlers.go

```diff
@@ -137,3 +139,20 @@ func HandleGetJSONBlockByHeight(fetcher ifaces.IBlockFetcher) func(w http.Respon
 		io.WriteString(w, string(marshalledBlock))
 	}
 }
+
+// GetBlockFromResponse will convert the body of a reponse handler and return a Block
+// Note - per the documentation on the response struct, the body is never nil
+// TODO how do we test the response is valid if the body is never nil?
```

Comment:
HTTP uses status codes. Please read up on them. 200 would be returned upon successful block retrieval

- [ ] remove TODO